packages/http-client/src/middleware/timeout.ts

```diff
@@ -0,0 +1,46 @@
+import { Context, Middleware, MiddlewareInOut } from './middleware-type';
+
+const defaultTimeout = 10000;
+
+/**
+ * Helper method to build a timout middleware.
+ * @internal
+ * @param timeoutValue - in miliseconds
```

Comment:
[req] Typos, order, full sentences and this is not a method ;)
```suggestion
 * @internal
 * Build a timeout middleware.
 * @param timeoutValue - Timeout in milliseconds.
```